Request/ParamFetcher.php

```diff
@@ -223,10 +223,15 @@ public function cleanParamWithRequirements(Param $config, $param, $strict)
                     $config->requirements
                 ),
             ));
+        } elseif(is_array($config->requirements) && isset($config->requirements["rule"]) && $config->requirements["error_message"]) {
```

Comment:
It is required because tests gives an fatal error
```json
PHP Fatal error:  Cannot use object of type Mock_Constraint_d66ec56b as array in /home/web/project/FOSRestBundle/Request/ParamFetcher.php on line 245
```